inst/components/scss/sidebar.scss

```diff
@@ -1,20 +1,27 @@
 $bslib-sidebar-bg: rgba(var(--bs-emphasis-color-rgb, 0,0,0), 0.05) !default;
 $bslib-sidebar-fg: var(--bs-emphasis-color, black) !default;
+$bslib-sidebar-toggle-bg: rgba(var(--bs-emphasis-color-rgb, 0,0,0), 0.1) !default;
 $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid var(--bs-card-border-color, #{$card-border-color}) !default;
 $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)), var(--bslib-sidebar-width, 250px));
 
 .bslib-sidebar-layout {
   --bslib-sidebar-transition-duration: 500ms;
+  --bslib-sidebar-transition-easing-x: cubic-bezier(0.8, 0.78, 0.22, 1.07);
   --bslib-sidebar-border: #{$bslib-sidebar-border};
   --bslib-sidebar-border-radius: var(--bs-border-radius);
   --bslib-sidebar-vert-border: #{$bslib-sidebar-border};
   --bslib-sidebar-bg: #{$bslib-sidebar-bg};
   --bslib-sidebar-fg: #{$bslib-sidebar-fg};
+  --bslib-sidebar-main-fg: var(--bs-card-color, var(--bs-body-color));
+  --bslib-sidebar-main-bg: var(--bs-card-bg, var(--bs-body-bg));
+  --bslib-sidebar-toggle-bg: #{$bslib-sidebar-toggle-bg};
   --bslib-sidebar-padding: calc(var(--bslib-spacer) * 1.5);
   --bslib-sidebar-icon-size: var(--bslib-spacer, 1rem);
-  --bslib-collapse-toggle-border: #{$bslib-sidebar-border};
-  --bslib-collapse-toggle-transform: 90deg;
-  --bslib-collapse-toggle-right-transform: -90deg;
+  // --bslib-collapse-toggle-border: ;
```

Comment:
```suggestion
```

Reply:
I think we should keep this as one step below actual documentation